ioctl/cmd/action/action.go

```diff
@@ -135,6 +135,9 @@ func registerWriteCommand(cmd *cobra.Command) {
 
 // gasPriceInRau returns the suggest gas price
 func gasPriceInRau() (*big.Int, error) {
+	if account.CryptoSm2 {
```

Comment:
we need to pay attention not to use this flag everywhere.

Reply:
Sure.